ui/src/app/pages/data/concept/concept-add-modal.tsx

```diff
@@ -26,13 +26,10 @@ const styles = reactStyles({
 });
 
 const filterConcepts = (concepts: any[], domain: Domain) => {
-  if (domain === Domain.PHYSICALMEASUREMENT) {
-    return concepts.filter(concept => concept.domainId === fp.capitalize(Domain[Domain.MEASUREMENT])
-      && concept.vocabularyId === CriteriaType[CriteriaType.PPI]);
-  } else if (domain === Domain.SURVEY) {
+  if (domain === Domain.SURVEY) {
     return concepts.filter(concept => !!concept.question);
   } else {
-    return concepts.filter(concept => concept.domainId === fp.capitalize(Domain[domain]));
+    return concepts.filter(concept => concept.domainId.replace(' ', '').toLowerCase() === Domain[domain].toLowerCase());
```

Comment:
At some point we should probably look at using Domain or DomainType but not both.

Reply:
Agreed. Looks like they both have at least one value that the other doesn't. Would be good to see where those are being used, if at all, and consolidate them.